db/migrate/20131030184435_create_houses.rb

```diff
@@ -1,8 +1,8 @@
 class CreateHouses < ActiveRecord::Migration
   def change
-    create_table :houses do |t|
+    create_table :houses do |f|
       f.string :name, null: false
-      f.string :img_url
+      f.string :image_url
```

Comment:
You do not need to change this. In fact, `img_url` is already used throughout the application.